hasjob/templates/stats.html.jinja2

```diff
@@ -21,7 +21,7 @@
       </div>
     </div>
 {% endblock %}
-{% block footerscripts %}
+{% block sheetscripts %}
```

Comment:
These templates should continue to use `footerscripts`. Only overwrite `sheetscripts` or `tablayoutscripts` in layout templates.

Reply:
In `stats.html.jinja`
  `{% block sheetscripts %}`
      `{% block footerscripts %}`
            .....
      `{% endblock %}`
  `{% endblock %}`
Would this be fine, @jace ?

Reply:
Using just `{% block footerscripts %}` should work.

Reply:
In `sheet.html.jinja2`, I have added 
`{% block layoutscripts %}`
  `{{ super() }}`
`{% endblock %}`
And used just `{% block footerscripts %}` in templates that extend `sheet.html.jinja2`